client/HelpCentrePage.ts

```diff
@@ -1,10 +1,16 @@
 import 'core-js/stable';
 import 'regenerator-runtime/runtime';
 import * as Sentry from '@sentry/browser';
-import 'ophan-tracker-js/build/ophan.manage-my-account';
 import { createRoot } from 'react-dom/client';
 import { HelpCentrePage } from './components/helpCentre/HelpCentrePage';
 
+// Initialize ophan on client side only
+if (typeof window !== 'undefined') {
```

Comment:
appreciate these dynamic sort of imports aren't nice. Ophan plan to implement SSR compatibility into `tracker-js` in the next version.